src/ray/common/ray_config_def.h

```diff
@@ -363,6 +363,10 @@ RAY_CONFIG(int, max_io_workers, 4)
 /// default. This value is not recommended to set beyond --object-store-memory.
 RAY_CONFIG(int64_t, min_spilling_size, 100 * 1024 * 1024)
 
+/// If set to less than 1.0, Ray will start spilling objects when existing objects
+/// take more than this percentage of the available memory.
+RAY_CONFIG(float, object_spilling_threshold, 1.0)
```

Comment:
Are we going to reduce this later?

Reply:
Probably not by default. I don't think preemptive spilling is always beneficial. It's just in the distributed sorting scenario we want to spill as early as possible.